source/browser.ts

```diff
@@ -539,16 +539,17 @@ ipc.answerMain('render-overlay-icon', (messageCount: number): {data: string; tex
 ipc.answerMain('render-native-emoji', (emoji: string): string => {
 	const canvas = document.createElement('canvas');
 	const context = canvas.getContext('2d')!;
+	const systemFont = is.linux ? 'emoji' : 'system-ui';
```

Comment:
```suggestion
	const systemFont = is.linux ? 'emoji, system-ui' : 'system-ui';
```

Would probably be smart to fas back on `system-ui` in case some Linux distro doesn't have `emoji`?